django/db/models/fields/__init__.py

```diff
@@ -144,6 +148,11 @@ def __init__(self, verbose_name=None, name=None, primary_key=False,
         self.remote_field = rel
         self.is_relation = self.remote_field is not None
         self.default = default
+        if db_default is not NOT_PROVIDED:
+            from django.db.models.expressions import Value
+            if not hasattr(db_default, 'resolve_expression'):
+                db_default = Value(db_default)
```

Comment:
Is this really necessary? Say that we avoided the `Value` wrapping here I assume `not hasattr(default, 'resolve_expression')` could be used in ` db_default_sql` instead of the instance check? See comment there.

Reply:
This means that, from now on, `db_default` is always an expression or the sentinel value `NOT_PROVIDED`. This is useful not only in `db_default_sql`, but also in `Field._check_db_default`.